i18n/web/en.ts

```diff
@@ -63,7 +63,7 @@ const en: WebTranslations = {
   residenceHistoryText:
     '<b>residence history</b> (number of years lived in Canada)',
   maritalStatusText: '<b>marital status</b>',
-  partnerText: `<b>partner</b> (if applicable):income (including their old age benefits), legal status, and residence history`,
+  partnerText: `<b>partner</b> (if applicable): income (including their old age benefits), legal status, and residence history`,
```

Comment:
good but may be changed soon - fine to keep in for now (though it is unrelated to this PR)